addons/website_slides/views/rating_rating_views.xml

```diff
@@ -109,21 +127,63 @@
         <field name="view_id" ref="rating_rating_view_kanban_slide_channel"/>
         <field name="help" type="html">
             <p class="o_view_nocontent_empty_folder">
-                There are no ratings for these courses at the moment
+                There are no reviews for these courses at the moment
             </p>
         </field>
     </record>
 
+    <record id="rating_rating_view_form" model="ir.ui.view">
+        <field name="name">rating.rating.form</field>
+        <field name="model">rating.rating</field>
+        <field name="active" eval="True"/>
+        <field name="inherit_id" ref="rating.rating_rating_view_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//form" position="replace">
+                <form string="Ratings" create="false">
+                    <sheet>
+                        <group>
+                            <group>
+                                <field name="partner_id"/>
+                                <field name="resource_ref" string="Course"/>
+                                <field name="feedback" attrs="{'invisible': [('feedback','=',False)]}"/>
+                            </group>
+                            <group>
+                                <field name="rating"/>
+                                <field name="create_date"/>
+                                <field name="is_internal"/>
+                            </group>
+                        </group>
+                    </sheet>
+                </form>
+            </xpath>
+        </field>
+    </record>
+
+    <record id="rating_rating_view_search" model="ir.ui.view">
+        <field name="name">rating.rating.search.inherit</field>
+        <field name="model">rating.rating</field>
+        <field name="inherit_id" ref="rating.rating_rating_view_search" />
+        <field name="arch" type="xml">
+            <xpath expr="//filter[@name='my_ratings']" position="replace"/>
+            <xpath expr="//filter[@name='responsible']" position="replace"/>
+            <xpath expr="//filter[@name='resource']" position="replace"/>
+        </field>
+    </record>
+
+
```

Comment:
![image](https://user-images.githubusercontent.com/33598078/134014508-9825d93a-6f40-4f3a-9fe4-69dda974dd6e.

First Link:
![image](https://user-images.githubusercontent.com/33598078/134014653-01c69f96-fbe1-4a76-abe7-de6888e6ce32.png)

Second Link:
![image](https://user-images.githubusercontent.com/33598078/134014561-5bb21177-3070-4fa5-9bd9-b388b4207745.png)

Reply:
I think you need to use
```
<field name="mode">primary</field>
```

And use a different ID (e.g.: `rating_rating_view_search_inherit_website_slides`), because currently you are removing those fields from the original search view.

See here, "My Rating" is hidden in Discuss when you install e-learning: https://drive.google.com/file/d/1nWjmVT-OFzsZqGGN9G1ilmHLJHlxgQmk/view?usp=sharing